src/lib/components/Pagination/index.tsx

```diff
@@ -1,205 +1,90 @@
-          <a
-            onClick={() => onPageChange(currentPage + 1 < totalPages ? currentPage + 1 : totalPages)}
+          <button
             className={classNames(
               'rounded-r-lg border border-gray-300 bg-white py-2 px-3 leading-tight text-gray-500 hover:bg-gray-100 hover:text-gray-700 dark:border-gray-700 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white',
               showIcon ? 'inline-flex' : '',
             )}
+            onClick={() => goToNextPage()}
           >
-            {!showIcon && 'Next'}
-            {showIcon && (
-              <>
-                Next
-                <BsArrowRight className="ml-3 h-5 w-5" />
-              </>
-            )}
-          </a>
+            Next
+            {showIcon && <HiChevronRight aria-hidden="true" className="h-5 w-5" />}
+          </button>
         </li>
       </ul>
     </nav>
   );
 };
 
-const renderNavigationGroup = (
-  currentPage: number,
-  totalPages: number,
-  onPageChange: (page: number) => void,
-  showIcon: boolean,
-) => {
-  return (
-    <div className="flex flex-col items-center">
-      <span className="text-sm text-gray-700 dark:text-gray-400">
-        Showing <span className="font-semibold text-gray-900 dark:text-white">{currentPage * 10 - 9}</span> to{' '}
-        <span className="font-semibold text-gray-900 dark:text-white">{currentPage * 10}</span> of{' '}
-        <span className="font-semibold text-gray-900 dark:text-white">{totalPages * 10}</span> Entries
-      </span>
-      <div className={classNames('xs:mt-0 mt-2 inline-flex', showIcon ? 'xs:mt-0 mt-2 inline-flex' : '')}>
-        <button
-          onClick={() => onPageChange(currentPage - 1 > 0 ? currentPage - 1 : 1)}
-          className={classNames(
-            'rounded-l bg-gray-800 py-2 px-4 text-sm font-medium text-white hover:bg-gray-900 dark:border-gray-700 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white',
-            showIcon ? 'inline-flex items-center' : '',
-          )}
-        >
-          {!showIcon && 'Prev'}
-          {showIcon && (
-            <>
-              <BsArrowLeft className="mr-3 h-5 w-5" />
-              Prev
-            </>
-          )}
-        </button>
-        <button
-          onClick={() => onPageChange(currentPage + 1 < totalPages ? currentPage + 1 : totalPages)}
-          className={classNames(
-            'rounded-r border-0 border-l border-gray-700 bg-gray-800 py-2 px-4 text-sm font-medium text-white hover:bg-gray-900 dark:border-gray-700 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white',
-            showIcon ? 'inline-flex items-center' : '',
-          )}
-        >
-          {!showIcon && 'Next'}
-          {showIcon && (
-            <>
-              Next
-              <BsArrowRight className="ml-3 h-5 w-5" />
-            </>
-          )}
-        </button>
-      </div>
-    </div>
-  );
-};
+export default Pagination;
```

Comment:
Please do not use `export default ...` for the main component so it can be called as `import { Pagination } from 'flowbite-react'`